### PR TITLE
Upgrade ember-cli, ember-cli-babel, ember-cli-htmlbars and ember-cli-shims

### PR DESCRIPTION
**Pull request by astronomersiva**
## What Changed & Why
Updates dependencies like ember-cli, ember-cli-babel, ember-cli-htmlbars and ember-cli-shims.

## People
@achambers , @cibernox

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "node tests/runner.js && ./node_modules/.bin/eslint index.js tests/**/*.js"
+    "test": "node tests/runner.js && node_modules/.bin/eslint index.js tests/**/*.js"
```
Comment (astronomersiva) [inline package.json]: tests were stuck without this change

File: tests/index-test.js [hunk 2]
```diff
@@ -7,12 +7,13 @@ var chaiAsPromised = require("chai-as-promised");
 chai.use(chaiAsPromised);
 var assert = chai.assert;
 var Project  = require('ember-cli/lib/models/project');
+var baseSubject = require('../index');
```
Comment (astronomersiva) [inline tests/index-test.js]: Refer https://github.com/babel/ember-cli-babel/issues/171#issuecomment-318916469

**Comment by astronomersiva** [comment]
Looking into why the tests are not getting terminated in travis-ci.

**Comment by astronomersiva** [comment]
@achambers @lukemelia, I would really appreciate if you take a look at this.

**Comment by lukemelia** [comment]
Thank you @astronomersiva. Your work was very helpful in preparing https://github.com/ember-cli-deploy/ember-cli-deploy-build/pull/47

**Pull request closed without merging**